C/plugins/common/include/omf.h

```diff
@@ -24,6 +24,11 @@
 #define OMF_TYPE_FLOAT		"number"
 #define OMF_TYPE_UNSUPPORTED	"unsupported"
 
+#define END_POINT_PIWEB_API  "p"
```

Comment:
It might be better to make this an enumerated type, that way you can avoid the expensive string comparisons.

Reply:
done, thanks.